app/static/css/admin.css

```diff
@@ -61,3 +64,14 @@ a.section:link, a.section:visited {
     width: auto;
     height: 70px;
 }
+
+
+.module h2, .module caption, .inline-group h2 {
+    margin: 0;
+    padding: 8px;
+    font-weight: 400;
+    font-size: 0.8125rem;
+    text-align: left;
+    background: var(--header-bg);
+    color: var(--primary-white);
+}
```

Comment:
This rule is mostly a duplication of the built-in one, with just `color` changed from `--header-link-color` to white. What about just overriding `--header-link-color` in the dark mode file? Then this duplication isn't needed.

If that doesn't work, please only override the part that needs overriding, otherwise we get to maintain the rest as well.

Reply:
I tried just changing in the dark mode, and it didn't work. So I kept the color to primary white, and removed the other duplicate code